src/hashtable.c

```diff
@@ -979,6 +980,25 @@ static void prefetchNextBucketEntries(iter *iter, bucket *current_bucket) {
     }
 }
 
+/* Prefetches the values associated with the entries in the given bucket by
+ * calling the entryPrefetchValue callback in the hashtableType, if defined. */
+static void prefetchBucketValues(bucket *b, hashtable *ht) {
+    if (!b->presence || ht->type->entryPrefetchValue != NULL) return;
```

Comment:
Having `ht->type->entryPrefetchValue != NULL` and setting `HASHTABLE_ITER_PREFETCH_VALUES` is a misconfiguration that isn't clearly flagged. Would be nice to have a runtime assert to make sure we are catching that.